tests/integration/route_test.py

```diff
@@ -807,6 +807,32 @@ def test_add_and_remove_ipv4_link_local_route(eth1_static_gateway_dns):
     assert IPV4_ADDRESS2 not in routes_output
 
 
+@pytest.mark.tier1
+def test_add_route_with_cwnd(eth1_up):
+    routes = [
+        {
+            Route.NEXT_HOP_INTERFACE: "eth1",
+            Route.DESTINATION: IPV4_TEST_NET1,
+            Route.NEXT_HOP_ADDRESS: IPV4_ADDRESS1,
+            Route.CWND: 20,
+        },
+        {
+            Route.NEXT_HOP_INTERFACE: "eth1",
+            Route.DESTINATION: IPV6_TEST_NET1,
+            Route.NEXT_HOP_ADDRESS: IPV6_GATEWAY1,
+            Route.CWND: 20,
```

Comment:
We may need integration tests changing cwnd value and remove routes with matching cwnd value.

Reply:
Since kernel and NM do not complain on setting cwnd value for non-unitcast route, maybe just ingore my comment on raise warning when setting the cwnd value with non-unicast route.